site/docs/components/flex-layout/usage.mdx

```diff
@@ -0,0 +1,38 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### When to use Flex Layout
+
+- When a complex layout cannot first be achieved using the Flow, Stack and Split layouts. However, if you are very familiar with [CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), you may choose to use Flex Layout from the start.
+
+- You want a layout that flows in a single dimension, where the dimension may change depending on other factors, such as viewport size. For example, in layouts where the direction and overflow behavior need to change depending on the viewport size or device.
+
+- For small-scale layout of components within a container or region within an application or webpage.
+
+### When not to use Flex Layout
+
+- When Flow, Stack or Split Layout can achieve your desired layout. These components are optimized for specific use cases in a constant direction.
+
+- When you need to present content over both rows and columns at the same time, such as a Dashboard or multi-column Form, use the Grid Layout.
+
+- For large-scale, page-level layout. Instead, use Grid or Border Layouts.
```

Comment:
@feliciackh should these point to Storybook for the time being?

Reply:
@JoanaMMoreira i think it should be ok to point them to site. Since the pages are all hidden for the time being we can probably just link them as if they currently already exist (that's what i've done for my components, also similarly done in som other components)